CREATE_WORKLOAD_GUIDE.md

```diff
@@ -15,32 +15,87 @@ $ opensearch-benchmark create-workload \
 --target-hosts="<CLUSTER ENDPOINT>" \
 --client-options="basic_auth_user:'<USERNAME>',basic_auth_password:'<PASSWORD>'" \
 --indices="<INDICES TO GENERATE WORKLOAD FROM>" \
+--number-of-docs="<INDEX NAME>:<NUMBER OF DOCUMENTS>" 
 --output-path="<LOCAL DIRECTORY PATH TO STORE WORKLOAD>"
+--custom-queries="<FILE PATH OF CUSTOM QUERIES TO ADD FOR THE WORKLOAD EXECUTION>"
+--concurrent
+--threads="<NUMBER OF THREADS TO RUN THE INDEX EXTRACTION>"
+--bsize="<BATCH SIZE OF EVERY INDEX EXTRACTION QUERY WITHIN THREADS>"
+--custom_dump_query="<FILE PATH OF THE CUSTOM DOCUMENT SEARCH QUERY FOR INDEX EXTRACTION>"
```

Comment:
Recommend renaming `--custom_dump_query` --> `--custom-dump-query`. 

Also, is this the `--custom-queries` parameter renamed? If so, I think it's better to keep it as `--custom-queries` over `--custom-dump-query` as the former is less verbose and implies that there is potentially one or more queries the user wishes to add.